modules/apps/digital-signature/digital-signature-impl/src/main/resources/content/Language.properties

```diff
@@ -3,6 +3,6 @@ api-account-id=API Account ID
 api-username=API Username
 category.digital-signature=Digital Signature
 digital-signature=Digital Signature
-digital-signature-configuration=Digital Signature Configuration
+digital-signature-configuration-name=Digital Signature Configuration
```

Comment:
I'll change this in next PR.
We need to change in DigitalSignatureConfiguration.java too.